### PR TITLE
image disappearance improved #6799

### LINKED ISSUE #6799: Make tag header images scale smoothly in mobile/desktop
We have these nice image headings for tag pages:

https://publiclab.org/tag/water-quality

They're based on this [Froala design block](https://www.froala.com/design-blocks) pattern:

image

However, ours aren't built the same way and don't smoothly/responsively transition between full-width and mobile sizes. There are some where the image is cut off, or messed up. Let's look back at the original design and code to try to standardize and fix this! 

Code for the original Froala block was:

```html
<section class="fdb-block pb-0" style="background-image: url(imgs/hero/purple.svg);">
  <div class="container">
    <div class="row text-left justify-content-end">
      <div class="col-12 col-md-6 col-xl-5">
        <div class="fdb-box rounded-bottom-0">
          <h2><strong>Design Blocks</strong></h2>
          <p class="lead">Far far away, behind the word mountains, far from the countries Vokalia and Consonantia, there live the blind texts. </p>

          <p class="lead mt-4"><a class="btn btn-outline-primary" href="https://www.froala.com">Read More</a></p>
        </div>
      </div>
    </div>
  </div>
</section>
```

I believe the full code can be found somewhere here:

https://github.com/froala/design-blocks/tree/dev/dist

Our code is here:

* https://github.com/publiclab/plots2/tree/master/app/views/tag/show.html.erb
* https://github.com/publiclab/plots2/tree/master/app/views/tag/show/

We'd love help fixing this!

### PR DESCRIPTION
**Pull request by NitinBhasneria**
Fixes #6799 (<=== Image disappearance improved between 560px and 768px)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!
Screenshot from 2020-02-07 18-51-33

### CODE REVIEW

File: app/views/tag/show/_header.html.erb [hunk 1]
```diff
@@ -140,4 +140,9 @@
 .bottom-right a {
   color: #fff;
 }
+
+@media (min-width: 576px){
```
Comment (VladimirMikulic) [inline app/views/tag/show/_header.html.erb]: The space is missing between `)` and `{`.

File: app/views/tag/show/_header.html.erb [hunk 2]
```diff
@@ -140,4 +140,9 @@
...
+.d-md-block {
+    display: block!important;
```
Comment (VladimirMikulic) [inline app/views/tag/show/_header.html.erb]: Here as well, `block` and `important` should be spaced out.

File: app/views/tag/show/_header.html.erb [hunk 3]
```diff
@@ -140,4 +140,9 @@
...
+}
```
Comment (VladimirMikulic) [inline app/views/tag/show/_header.html.erb]: You didn't close your `@media` rule :)

File: app/views/tag/show/_header.html.erb [hunk 4]
```diff
@@ -140,4 +140,10 @@
 .bottom-right a {
   color: #fff;
 }
+
+@media (min-width: 576px) {
+.d-md-block {
```
Comment (VladimirMikulic) [inline app/views/tag/show/_header.html.erb]: Properly indent the `.d-md-block` rule.

Comment (NitinBhasneria) [reply app/views/tag/show/_header.html.erb]: @VladimirMikulic I dont know the .d-md-block rules can you please guide me.

Comment (VladimirMikulic) [reply app/views/tag/show/_header.html.erb]: Alright, this is your CSS:
```css
@media (min-width: 576px) {
.d-md-block {
    display: block !important;
}
}
```

This is how it should be formatted:

```css
@media (min-width: 576px) {
    .d-md-block {
        display: block !important;
    }
}
```

**Comment by NitinBhasneria** [comment]
@VladimirMikulic Can you tell why is this error??

**Comment by VladimirMikulic** [comment]
Sometimes, tests fail unexpectedly. Unfortunately that's all I can say because I am not an admin. To resolve this, restart Travis by force pushing.

**Comment by VladimirMikulic** [comment]
:rocket:

**Comment by jywarren** [comment]
Thank you!!

**Pull request merged**